src/vs/workbench/browser/web.main.ts

```diff
@@ -140,6 +145,9 @@ export class BrowserMain extends Disposable {
 					}
 				},
 				window: {
```

Comment:
I feel a better top level domain for this would be `logger` or `diagnostics`. I associate `window` more with UI things.